addons/sale_project/models/project.py

```diff
@@ -84,6 +84,15 @@ def _compute_sale_order_count(self):
         for project in self:
             project.sale_order_count = len(sale_order_items_per_project_id.get(project.id, self.env['sale.order.line']).order_id)
 
+    def _compute_invoice_count(self):
+        query = self.env['account.move.line']._search([('move_id.move_type', 'in', ['out_invoice', 'out_refund'])])
+        query.add_where('analytic_distribution ? %s', [str(self.analytic_account_id.id)])
+        query.order = None
+        query_string, query_param = query.select('COUNT(DISTINCT move_id)')
+        self._cr.execute(query_string, query_param)
+        count = self._cr.fetchone()[0]
```

Comment:
a compute method could contained more than one record in `self` recordset. 😬 
Can you adapt your code to manage the case where `self` contains more than one record? Thanks in advance!

Reply:
Damn, sorry, I always forget about that. It's fixed :)